### PR TITLE
Updated repo card design to be more inline with mocks

### PR DESCRIPTION
**Pull request by raulkele**
Signed-off-by: Raul Kele <raulkeleblk@gmail.com>

**What type of PR is this?**

**Which issue does this PR fix**:
Closes Issue #26 

**What does this PR do / Why do we need it**:
Update RepoCard, PreviewCard styling to be more inline with mocks

**Does this PR introduce any user-facing change?**:

```release-note
- Updated RepoCard component styles
- Updated PreviewCard component styles
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by njiandan1255 — CHANGES_REQUESTED** [review]

File: src/components/RepoCard.jsx [hunk 1]
```diff
@@ -55,6 +56,10 @@ const useStyles = makeStyles(() => ({
   content: {
      textAlign: "left",
      color: "#606060",
+     maxHeight:"148px"
```
Comment (njiandan1255) [inline src/components/RepoCard.jsx:59]: try to use % or em/rem instead of hard coding pixels

File: src/components/PreviewCard.jsx [hunk 2]
```diff
@@ -17,28 +17,30 @@ const randomIntFromInterval = (min, max) => {
...
-      flexGrow:0,
-      order:0,
-      width:"100%",
+    marginBottom: 2,
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    background: "#FFFFFF",
+    boxShadow: "0px 5px 10px rgba(131, 131, 131, 0.08)",
+    borderRadius: "24px",
```
Comment (njiandan1255) [inline src/components/PreviewCard.jsx:32]: try to use relative units, i.e. em/rem or %

**Review by njiandan1255 — CHANGES_REQUESTED** [review]

File: src/components/PreviewCard.jsx (hunk 2 above)
Comment (raulkele) [reply src/components/PreviewCard.jsx]: changed all px measurements to rem

File: src/components/RepoCard.jsx (hunk 1 above)
Comment (raulkele) [reply src/components/RepoCard.jsx]: changed all px measurements to rem

**Review by njiandan1255 — APPROVED** [review]

**Review by njiandan1255 — APPROVED** [review]

**Review by njiandan1255 — APPROVED** [review]

**Pull request merged**